### ISSUE TITLE
mouse cursor size is not adjustable

### ISSUE THREAD

**Issue by pvanek** [lxqt/lxqt#953]
It's quite critical for modern high-resolution displays. KDE and GNOME offer this option.

**Comment by comodoro** [comment]
I am using 0.11.1, which is current for current Ubuntu apt. Is 0.12 going to main repository soon, or is there a workaround? There is .config/lxsession/lxqt/desktop.conf with [Mouse] section, but I cannot find any documentation about it anywhere.

**Comment by stefonarch** [comment]
Files in lxsession and desktop.conf  (the latter maybe existed once time ago) are not part of LXQt, maybe it's ubuntu's work. As far as I know there is no option to modify cursor size, but you can choose a cursor theme with big cursors.

**Comment by comodoro** [comment]
OK, I tried to change the theme in the lxqt-config app, but there are only four and none is big enough. I suppose any LXQT configuration files (where are they?) will not help?

**Comment by tsujan** [comment]
@comodoro This is a valid feature request and is marked as so. No option for changing the cursor size exists in LXQt settings yet but there other ways to do so; for example, see https://wiki.archlinux.org/index.php/Cursor_themes

**Comment by comodoro** [comment]
Very helpful link, thanks! In addition to copying the cursor theme to
~/.icons, in LXQT it is also necessary to set it in lxqt-config for it to
be used on LXQT desktop elements. I have successfully added the theme
https://www.gnome-look.org/content/show.php/Large+Mouse+Cursors?content=140787

2017-10-03 13:55 GMT+02:00 tsujan <notifications@github.com>:

**Comment by tsujan** [comment]
So, you used a large cursor theme. Sooner or later, a feature should be added to LXQt settings for changing the cursor size *when the theme supports multiple sizes* -- which is the subject of this report.

**Comment by sroracle** [comment]
As a workaround, setting the XCURSOR_SIZE environment variable works. It should be noted that the Xcursor.size resource in ~/.Xresources or similar is ignored however. This issue should be looked at as out-of-box behavior on HiDPI systems is to have the cursor as small as possible.

**Comment by scott092707** [comment]
>It should be noted that the Xcursor.size resource in ~/.Xresources or similar is ignored however.
I have independently discovered this...
(by the way - I use Debian, and it has ~/.Xdefaults, but a recent version of Lubuntu apparently has ~/.Xresources - Should I assume that which is present is NOT an LXQt decision but that of the distro?)
Using the _SIZE environment variable does work.

I don't know the command xrdb, but it came up in my investigation, and I note that although:
scott@ASUS-PRIME-B350M-A-CSM:~$ cat ~/.icons/default/index.theme
# Written by lxqt-config-appearance
[Icon Theme]
Name=Default
Comment=Default cursor theme
Inherits=oxy-red-argentina
scott@ASUS-PRIME-B350M-A-CSM:~$ cat .Xdefaults

Xcursor.theme: oxy-red-argentina
Xcursor.size: 48

Note that:
scott@ASUS-PRIME-B350M-A-CSM:~$ xrdb -query
*customization: -color
Xcursor.size:   18
Xcursor.theme:  oxy-red-argentina
Xcursor.theme_core:     true

This DESPITE having used the environment variable.

WHERE is xrdb getting the "18" ?
Perhaps if we could fix that, the environment variable would be unnecessary...

Next question, is there some command/app that, when pointed at the theme directory, would then output all the included/possible sizes?
Although I love my red cursor (especially at the current 48 size, on a 49" screen), I had to guess a number of times (using the environment variable as a prefix to the fpad command), before I could find a size that worked.  Nothing in the filenames of the cursor theme indicates anything about size.

**Comment by tsujan** [comment]
Not valid anymore; should have been closed years ago.
